# Asturian translation for update-manager
# Copyright (c) 2008 Rosetta Contributors and Canonical Ltd 2008
# This file is distributed under the same license as the update-manager package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2008.
#
#: ../DistUpgrade/DistUpgradeController.py:1075
msgid ""
msgstr ""
"Project-Id-Version: update-manager\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2021-04-13 11:08-0700\n"
"PO-Revision-Date: 2019-06-18 17:54+0000\n"
"Last-Translator: ivarela <ivarela@ubuntu.com>\n"
"Language-Team: Asturian <ast@li.org>\n"
"Language: ast\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Launchpad-Export-Date: 2019-11-15 04:46+0000\n"
"X-Generator: Launchpad (build c597c3229eb023b1e626162d5947141bf7befb13)\n"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:215 ../DistUpgrade/distro.py:446
#, python-format
msgid "Server for %s"
msgstr "Sirvidores pa %s"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:233 ../DistUpgrade/distro.py:239
#: ../DistUpgrade/distro.py:255
msgid "Main server"
msgstr "Sirvidor principal"

#: ../DistUpgrade/distro.py:259
msgid "Custom servers"
msgstr "Sirvidores personalizaos"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:151
msgid "Could not calculate sources.list entry"
msgstr "Nun pudo calculase la entrada sources.list"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:260
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr ""
"Nun puede llocalizase dengún paquete, seique nun ye un discu d'Ubuntu o nun "
"ye l'arquiteutura correuta."

#: ../DistUpgrade/DistUpgradeAptCdrom.py:310
msgid "Failed to add the CD"
msgstr "Falló amestar el CD"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:311
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"Asocedió un erru al amestar el CD; Torgóse l'anovamientu. Por favor, informe "
"d'esto como un fallu si esti ye un CD válidu d'Ubuntu.\n"
"\n"
"El mensax d'error foi:\n"
"«%s»"

#: ../DistUpgrade/DistUpgradeCache.py:182
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Desaniciar paquete en mal estáu"
msgstr[1] "Desaniciar paquetes en mal estáu"

#: ../DistUpgrade/DistUpgradeCache.py:185
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"El paquete «%s» ta nun estáu inconsistente y hai de reinstalalu, pero nun "
"s'alcuentra en dengún repositoriu. ¿Quies desinstalar esti paquete agora pa "
"continuar?"
msgstr[1] ""
"Los paquetes «%s» tán nun estáu inconsistente y hai de reinstalalos, pero "
"nun s'alcuentren en dengún repositoriu. ¿Quies desinstalar estos paquete "
"agora pa continuar?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:301
msgid "The server may be overloaded"
msgstr "El sirvidor puede tar sobrocargáu"

#: ../DistUpgrade/DistUpgradeCache.py:414
msgid "Broken packages"
msgstr "Paquetes frayaos"

#: ../DistUpgrade/DistUpgradeCache.py:415
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"El to sistema contién paquetes frayaos que nun pueden iguase con esti "
"software. Por favor ígualo enantes d'usar synaptic o apt-get"

#: ../DistUpgrade/DistUpgradeCache.py:660
#, fuzzy, python-format
msgid "Broken packages after upgrade: %s"
msgstr "Paquetes frayaos"

#. the most likely problem is the 3rd party pkgs so don't address
#. foreignPkgs and devRelease being True
#: ../DistUpgrade/DistUpgradeCache.py:669
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
" "
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:672
msgid ""
"This was likely caused by:\n"
" * Unofficial software packages not provided by Ubuntu\n"
"Please use the tool 'ppa-purge' from the ppa-purge \n"
"package to remove software from a Launchpad PPA and \n"
"try the upgrade again.\n"
"\n"
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:680
msgid ""
"This was caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
"This is most likely a transient problem, \n"
"please try again later.\n"
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:687
msgid "This is most likely a transient problem, please try again later."
msgstr "Dablemente seya un problema transitoriu, téntelo otra vegada más sero."

#: ../DistUpgrade/DistUpgradeCache.py:690
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug ubuntu-release-upgrader-core' in a terminal. "
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:692
msgid ""
"If you want to investigate this yourself the log files in '/var/log/dist-"
"upgrade' will contain details about the upgrade. Specifically, look at 'main."
"log' and 'apt.log'."
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:698
msgid "Could not calculate the upgrade"
msgstr "Nun pue calculase l'anovamientu de versión"

#: ../DistUpgrade/DistUpgradeCache.py:757
msgid "Error authenticating some packages"
msgstr "Error autentificando dalgunos paquetes"

#: ../DistUpgrade/DistUpgradeCache.py:758
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"Nun se foi quien a autenticar dalgunos paquetes. Esto pue debese a un "
"problema transitoriu na rede. Pruebe otra vegada más sero. Vea abaxo una "
"llista de los paquetes non autenticaos."

#: ../DistUpgrade/DistUpgradeCache.py:779
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr ""
"El paquete '%s' ta conseñáu pa desaniciar, pero ta na llista de non "
"desaniciables."

#: ../DistUpgrade/DistUpgradeCache.py:785
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "El paquete esencial '%s' ta conseñáu pa desaniciar."

#: ../DistUpgrade/DistUpgradeCache.py:794
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr "Intentando instalar la versión prohibida «%s»"

#: ../DistUpgrade/DistUpgradeCache.py:916
#, python-format
msgid "Can't install '%s'"
msgstr "Nun puede instalase '%s'"

#: ../DistUpgrade/DistUpgradeCache.py:917
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
msgstr ""
"Nun foi dable instalar un paquete requeríu. Informa d'esto como un fallu "
"usando «ubuntu-bug ubuntu-release-upgrader-core» nuna terminal."

#: ../DistUpgrade/DistUpgradeCache.py:928
msgid "Can't guess meta-package"
msgstr "Nun pudo determinase'l meta-paquete"

#: ../DistUpgrade/DistUpgradeCache.py:929
#, python-format
msgid ""
"Your system does not contain a %s or %s package and it was not possible to "
"detect which version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"El sistema nun contién un paquete %s o %s y nun se pudo detectar qué versión "
"d'Ubuntu ta executando.\n"
" Instale unu de los paquetes anteriores usando synaptic o apt-get antes de "
"siguir."

#: ../DistUpgrade/DistUpgradeController.py:113
msgid "Reading cache"
msgstr "Lleendo cache"

#: ../DistUpgrade/DistUpgradeController.py:225
msgid "Unable to get exclusive lock"
msgstr "Nun pudo obtenese un bloquéu esclusivu"

#: ../DistUpgrade/DistUpgradeController.py:226
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Esto normalmente quier dicir que yá ta executándose otra aplicación de "
"xestión de paquetes (como apt-get o aptitude). Por favor, pieslla esa "
"aplicación primero."

#: ../DistUpgrade/DistUpgradeController.py:284
msgid "Upgrading over remote connection not supported"
msgstr "Nun hai sofitu p'anovar per conexón remota"

#: ../DistUpgrade/DistUpgradeController.py:285
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"Tas executando l'anovamientu sobre una conexón ssh remota con una interface "
"d'usuariu que nun lo permite. Intenta anovar en mou testu con «do-release-"
"upgrade».\n"
"\n"
"L'anovamientu va parase agora. Inténtalo ensin ssh."

#: ../DistUpgrade/DistUpgradeController.py:299
msgid "Continue running under SSH?"
msgstr "¿Continuar executando baxo SSH?"

#: ../DistUpgrade/DistUpgradeController.py:300
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"Esta sesión paez tar executándose baxo ssh. Nun ye recomendable facer agora "
"un anovamientu sobre ssh, porque en caso de fallu faise bien abegoso la "
"recuperación.\n"
"\n"
"Si sigues, aniciaráse un degorriu ssh adicional nel puertu «%s».\n"
"¿Quies siguir?"

#: ../DistUpgrade/DistUpgradeController.py:314
msgid "Starting additional sshd"
msgstr "Aniciando sshd adicional"

#: ../DistUpgrade/DistUpgradeController.py:315
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"Pa facer más fácil la recuperación en casu de fallu, aniciaráse un sshd "
"estra nel puertu «%s». Si daqué va mal col ssh n'execución, entá podrás "
"coneutate al estra.\n"

#: ../DistUpgrade/DistUpgradeController.py:323
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""
"Si executes un tornafueos, pues necesitar abrir esti puertu temporalmente. "
"Como esto ye potencialmente peligroso, nun se fai automáticamente. Pues "
"abrir el puertu con:\n"
"«%s»"

#: ../DistUpgrade/DistUpgradeController.py:401
#: ../DistUpgrade/DistUpgradeController.py:425
msgid "Can not upgrade"
msgstr "Nun se pue anovar"

#: ../DistUpgrade/DistUpgradeController.py:402
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "Esta ferramienta nun soporta anovamientos de «%s» a «%s»."

#: ../DistUpgrade/DistUpgradeController.py:426
msgid ""
"Your python3 install is corrupted. Please fix the '/usr/bin/python3' symlink."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:455
#, python-format
msgid "Can not write to '%s'"
msgstr "Nun pue escribise en «%s»"

#: ../DistUpgrade/DistUpgradeController.py:456
#, python-format
msgid ""
"Its not possible to write to the system directory '%s' on your system. The "
"upgrade can not continue.\n"
"Please make sure that the system directory is writable."
msgstr ""
"Nun ye dable escribir el direutoriu de sistema «%s» nel to sistema. "
"L'anovamientu nun pue continuar.\n"
"Asegúrate de que'l direutoriu de sistema permite escribir."

#: ../DistUpgrade/DistUpgradeController.py:467
msgid "Include latest updates from the Internet?"
msgstr "¿Incluyir los caberos anovamientos dende Internet?"

#: ../DistUpgrade/DistUpgradeController.py:468
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"El sistema d'anovamientu a una nueva versión puede usar Internet pa baxar "
"automáticamente los anovamientos más recientes ya instalalos durante'l "
"procesu. Si dispones d'una conexón a la rede, esto encamiéntase enforma.\n"
"\n"
"L'anovamientu llevará más tiempu, pero en terminando, el sistema tará "
"dafechu anováu. Puedes escoyer nun facelo, pero vas tener d'instalar los "
"nuevos anovamientos darréu de pasar a la nueva versión.\n"
"Si respuendes «non» agora, la rede nun s'usará pa nada."

#: ../DistUpgrade/DistUpgradeController.py:646
#, python-format
msgid "Not for humans during development stage of release %s"
msgstr ""
"Nun ta destináu a usuarios mientres la etapa de desendolcu de la versión %s"

#: ../DistUpgrade/DistUpgradeController.py:747
#, python-format
msgid "disabled on upgrade to %s"
msgstr "deshabilitáu nel anovamientu a %s"

#: ../DistUpgrade/DistUpgradeController.py:806
msgid "No valid sources.list entry found"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:807
#, python-format
msgid ""
"While scanning your repository information no entry about %s could be "
"found.\n"
"\n"
"An upgrade might not succeed.\n"
"\n"
"Do you want to continue anyway?"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:817
msgid "No valid mirror found"
msgstr "Nun s'atopó un espeyu válidu"

#: ../DistUpgrade/DistUpgradeController.py:818
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run an internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""
"Mientres s'analizaba la información de los repositorios, nun s'atopó una "
"entrada de sirvidor espeyu pal anovamientu. Esto pue asoceder si tienes el "
"to propiu sirvidor internu o si la información nun ta anovada.\n"
"\n"
"¿Quies reescribir el ficheru «sources.list» de toes formes? Si escueyes «Sí» "
"van anovase toles entraes «%s» a «%s».\n"
"Si escueyes «Non», va encaboxase l'anovamientu."

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:838
msgid "Generate default sources?"
msgstr "¿Xenerar fontes predeterminaes?"

#: ../DistUpgrade/DistUpgradeController.py:839
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"Dempués de desaminar el to 'sources.list' nun s'alcontró denguna entrada "
"válida pa '%s'.\n"
"\n"
"Tendríen d'amestase les entraes predeterminaes pa '%s'? Si respuendes 'Non', "
"encaboxaráse l'anovamientu."

#: ../DistUpgrade/DistUpgradeController.py:874
#: ../DistUpgrade/DistUpgradeController.py:880
msgid "Repository information invalid"
msgstr "Información del repositoriu non válida"

#: ../DistUpgrade/DistUpgradeController.py:875
msgid ""
"Upgrading the repository information resulted in a invalid file so a bug "
"reporting process is being started."
msgstr ""
"L'anovamientu de la información del repositoriu dio como resultáu un ficheru "
"inválidu polo que ta aniciándose un procesu de notificación d'errores."

#: ../DistUpgrade/DistUpgradeController.py:881
msgid ""
"Upgrading the repository information resulted in a invalid file. To report a "
"bug install apport and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""
"L'anovamientu de la información del repositoriu produxo un ficheru "
"incorreutu. Pa informar d'un fallu instale apport y execute «apport-bug "
"ubuntu-release-upgrader»."

#: ../DistUpgrade/DistUpgradeController.py:890
msgid "Third party sources disabled"
msgstr "Desactivar fontes de terceres partes"

#: ../DistUpgrade/DistUpgradeController.py:891
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"Desautiváronse delles entraes de terceros nel to «sources.list». Pues tornar "
"a activales tres l'anovamientu cola ferramienta «Oríxenes del software», o "
"col xestor de paquetes."

#: ../DistUpgrade/DistUpgradeController.py:933
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Paquete nun estáu inconsistente"
msgstr[1] "Paquetes nun estáu inconsistente"

#: ../DistUpgrade/DistUpgradeController.py:936
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"El paquete «%s» ta nun estáu inconsistente y hai de reinstalalu, pero nun "
"s'alcuentra en dengún repositoriu. Por favor, reinstale'l paquete "
"manualmente o desinstálelu del sistema"
msgstr[1] ""
"Los paquetes «%s» tán nun estáu inconsistente y hai de reinstalalos, pero "
"nun s'alcuentren en dengún repositoriu. Por favor, reinstale los paquetes "
"manualmente o desinstálelos del sistema"

#: ../DistUpgrade/DistUpgradeController.py:999
msgid "Error during update"
msgstr "Fallu durante l'anovamientu"

#: ../DistUpgrade/DistUpgradeController.py:1000
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"Hebo un fallu durante l'anovamientu. Davezu suel ser un tipu de problema na "
"rede, polo qu'encamentámoste que compruebes la conexón de rede y tornes a "
"intentalo."

#: ../DistUpgrade/DistUpgradeController.py:1043
#: ../DistUpgrade/DistUpgradeController.py:1048
msgid "EFI System Partition (ESP) not usable"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1044
msgid ""
"Your EFI System Partition (ESP) is not mounted at /boot/efi. Please ensure "
"that it is properly configured and try again."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1049
msgid ""
"The EFI System Partition (ESP) mounted at /boot/efi is not writable. Please "
"mount this partition read-write and try again."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1056
msgid "Not enough free disk space"
msgstr "Nun hai suficiente espaciu llibre en discu"

#. TRANSLATORS: you can change the order of the sentence,
#. make sure to keep all {str_*} string untranslated.
#: ../DistUpgrade/DistUpgradeController.py:1059
#, python-brace-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of {str_total} free space "
"on disk '{str_dir}'. Please free at least an additional {str_needed} of disk "
"space on '{str_dir}'. {str_remedy}"
msgstr ""

#. specific ways to resolve lack of free space
#: ../DistUpgrade/DistUpgradeController.py:1064
msgid ""
"Remove temporary packages of former installations using 'sudo apt clean'."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1066
msgid ""
"You can remove old kernels using 'sudo apt autoremove' and you could also "
"set COMPRESS=xz in /etc/initramfs-tools/initramfs.conf to reduce the size of "
"your initramfs."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1071
msgid ""
"Empty your trash and remove temporary packages of former installations using "
"'sudo apt-get clean'."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1074
msgid "Reboot to clean up files in /tmp."
msgstr ""

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:1113
#: ../DistUpgrade/DistUpgradeController.py:1997
#: ../DistUpgrade/DistUpgradeController.py:2047
msgid "Calculating the changes"
msgstr "Calculando los cambeos"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:1157
msgid "Do you want to start the upgrade?"
msgstr "¿Quies aniciar l'anovamientu?"

#: ../DistUpgrade/DistUpgradeController.py:1176
#, python-format
msgid ""
"Livepatch security updates are not available for Ubuntu %s. If you upgrade, "
"Livepatch will turn off."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1248
msgid "Upgrade canceled"
msgstr "Anovamientu encaboxáu"

#: ../DistUpgrade/DistUpgradeController.py:1249
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr ""
"L'anovamientu va encaboxase agora y el sistema va volver al so estáu "
"orixinal. Pues reanudar l'anovamientu más sero."

#: ../DistUpgrade/DistUpgradeController.py:1255
#: ../DistUpgrade/DistUpgradeController.py:1400
msgid "Could not download the upgrades"
msgstr "Nun se puede descargar les actualizaciones"

#: ../DistUpgrade/DistUpgradeController.py:1256
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr ""
"Encaboxóse l'anovamientu. Comprueba la conexón a Internet o los medios "
"d'instalación y vuelvi a intentalo. Van caltenese tolos ficheros descargaos "
"hasta'l momentu."

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1349
#: ../DistUpgrade/DistUpgradeController.py:1388
#: ../DistUpgrade/DistUpgradeController.py:1504
msgid "Error during commit"
msgstr "Error durante la confirmación"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1351
#: ../DistUpgrade/DistUpgradeController.py:1390
#: ../DistUpgrade/DistUpgradeController.py:1551
msgid "Restoring original system state"
msgstr "Restaurando al estau del sistema orixinal"

#: ../DistUpgrade/DistUpgradeController.py:1352
#: ../DistUpgrade/DistUpgradeController.py:1367
#: ../DistUpgrade/DistUpgradeController.py:1391
msgid "Could not install the upgrades"
msgstr "Nun pudieron instalase los anovamientos"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1357
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"Encaboxóse l'anovamientu. Pue que'l sistema quedare nun estáu non usable. "
"Agora, va facese una recuperación (dpkg --configure -a)."

#: ../DistUpgrade/DistUpgradeController.py:1362
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug in a browser at http://bugs.launchpad.net/ubuntu/"
"+source/ubuntu-release-upgrader/+filebug and attach the files in /var/log/"
"dist-upgrade/ to the bug report.\n"
"%s"
msgstr ""
"\n"
"\n"
"Informa d'esti fallu usando un navegador pa dir a http://bugs.launchpad.net/"
"ubuntu/+source/ubuntu-release-upgrader/+filebug y axunta los ficheros en /"
"var/log/dist-upgrade/ al informe.\n"
"%s"

#: ../DistUpgrade/DistUpgradeController.py:1401
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"Encaboxóse l'anovamientu. Por favor, comprueba la conexón a Internet o el "
"sofitu d'instalación y vuelvi a intentalo. "

#: ../DistUpgrade/DistUpgradeController.py:1429
msgid "Searching for obsolete software"
msgstr "Buscando software obsoletu"

#: ../DistUpgrade/DistUpgradeController.py:1492
msgid "Remove obsolete packages?"
msgstr "¿Desaniciar paquetes obsoletos?"

#: ../DistUpgrade/DistUpgradeController.py:1493
#: ../data/gtkbuilder/DistUpgrade.ui.h:8
msgid "_Keep"
msgstr "_Caltener"

#: ../DistUpgrade/DistUpgradeController.py:1493
msgid "_Remove"
msgstr "_Esborrar"

#: ../DistUpgrade/DistUpgradeController.py:1505
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr ""
"Hebo un fallu durante'l llimpiáu. Por favor, llea'l mensax siguiente pa más "
"información. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1589
msgid "Required depends is not installed"
msgstr "Dependencia requería nun ta instalada"

#: ../DistUpgrade/DistUpgradeController.py:1590
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "La dependencia requería «%s» nun ta instalada. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1857
#: ../DistUpgrade/DistUpgradeController.py:1947
msgid "Checking package manager"
msgstr "Comprobando'l xestor de paquetes"

#: ../DistUpgrade/DistUpgradeController.py:1863
#: ../DistUpgrade/DistUpgradeController.py:1869
msgid "Preparing the upgrade failed"
msgstr "Falló la tresna del anovamientu"

#: ../DistUpgrade/DistUpgradeController.py:1864
msgid ""
"Preparing the system for the upgrade failed so a bug reporting process is "
"being started."
msgstr ""
"Hebo un fallu al preparar el sistema pal anovamientu poro, ta arrancando un "
"procesu de notificación de fallos."

#: ../DistUpgrade/DistUpgradeController.py:1870
msgid ""
"Preparing the system for the upgrade failed. To report a bug install apport "
"and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""
"Falló la preparación del sistema pal anovamientu. Pa informar d'un fallu "
"instale apport y execute «apport-bug ubuntu-release-upgrader»."

#: ../DistUpgrade/DistUpgradeController.py:1887
#: ../DistUpgrade/DistUpgradeController.py:1897
msgid "Getting upgrade prerequisites failed"
msgstr "Falló la tresna del anovamientu"

#: ../DistUpgrade/DistUpgradeController.py:1888
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Additionally, a bug reporting process is being started."
msgstr ""
"El sistema nun foi a obtener los requisitos previos pal anovamientu. "
"L'anovamientu va encaboxase agora y va restaurase l'estáu orixinal del "
"sistema.\n"
"\n"
"Adicionalmente, ta aniciándose un procesu de notificación d'errores."

#: ../DistUpgrade/DistUpgradeController.py:1898
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"To report a bug install apport and then execute 'apport-bug ubuntu-release-"
"upgrader'."
msgstr ""
"El sistema nun pudo cumplir los requisitos previos pal anovamientu. "
"L'anovamientu torgaráse agora ya'l sistema devolveráse al so estáu "
"orixinal.\n"
"\n"
"Pa informar d'un fallu instale apport y execute «apport-bug ubuntu-release-"
"upgrader»."

#: ../DistUpgrade/DistUpgradeController.py:1931
msgid "Updating repository information"
msgstr "Anovando información del repositoriu"

#: ../DistUpgrade/DistUpgradeController.py:1938
msgid "Failed to add the cdrom"
msgstr "Fallu al amestar el CDROM"

#: ../DistUpgrade/DistUpgradeController.py:1939
msgid "Sorry, adding the cdrom was not successful."
msgstr "Sentímoslo, nun pudo amestase'l CDROM"

#: ../DistUpgrade/DistUpgradeController.py:1974
msgid "Invalid package information"
msgstr "Información del paquete nun válida"

#: ../DistUpgrade/DistUpgradeController.py:1975
#, python-format
msgid ""
"After updating your package information, the essential package '%s' could "
"not be located. This may be because you have no official mirrors listed in "
"your software sources, or because of excessive load on the mirror you are "
"using. See /etc/apt/sources.list for the current list of configured software "
"sources.\n"
"In the case of an overloaded mirror, you may want to try the upgrade again "
"later."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:2004
#: ../DistUpgrade/DistUpgradeController.py:2103
msgid "Fetching"
msgstr "Descargando"

#: ../DistUpgrade/DistUpgradeController.py:2011
#: ../DistUpgrade/DistUpgradeController.py:2032
#: ../DistUpgrade/DistUpgradeController.py:2053
#: ../DistUpgrade/DistUpgradeController.py:2107
msgid "Upgrading"
msgstr "Anovando"

#: ../DistUpgrade/DistUpgradeController.py:2013
msgid "Upgrade infeasible"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:2014
msgid ""
"The upgrade could not be completed, there were errors during the upgrade "
"process."
msgstr ""

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:2035
msgid "Upgrade incomplete"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:2036
msgid ""
"The upgrade has partially completed but there were errors during the upgrade "
"process."
msgstr ""

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:2058
#: ../DistUpgrade/DistUpgradeController.py:2109
#: ../DistUpgrade/DistUpgradeController.py:2115
#: ../DistUpgrade/DistUpgradeController.py:2126
msgid "Upgrade complete"
msgstr "Completóse l'anovamientu"

#: ../DistUpgrade/DistUpgradeController.py:2059
#: ../DistUpgrade/DistUpgradeController.py:2110
#: ../DistUpgrade/DistUpgradeController.py:2116
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr "L'anovamientu completóse pero hebo fallos durante'l procesu."

#: ../DistUpgrade/DistUpgradeController.py:2078
msgid "System upgrade is complete."
msgstr "anovamientu del sistema completáu"

#: ../DistUpgrade/DistUpgradeController.py:2127
msgid "The partial upgrade was completed."
msgstr "L'anovamientu parcial completóse."

#: ../DistUpgrade/DistUpgradeController.py:2155
msgid "Lock screen disabled"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:2156
msgid ""
"Your lock screen has been disabled and will remain disabled until you reboot."
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcher.py:107
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:170
msgid "Could not find the release notes"
msgstr "Nun s'alcontraron les notes de publicación"

#: ../DistUpgrade/DistUpgradeFetcher.py:108
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:171
msgid "The server may be overloaded. "
msgstr "Puede que'l sirvidor tea sobrocargáu. "

#: ../DistUpgrade/DistUpgradeFetcher.py:120
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:175
msgid "Could not download the release notes"
msgstr "Nun pudieron descargase les notes d'espublización"

#: ../DistUpgrade/DistUpgradeFetcher.py:121
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:176
msgid "Please check your internet connection."
msgstr "Por favor, comprueba la conexón a Internet."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:73
#, python-format
msgid "authenticate '%(file)s' against '%(signature)s' "
msgstr "autentificar «%(file)s» escontra «%(signature)s» "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:99
#, python-format
msgid "extracting '%s'"
msgstr "estrayendo «%s»"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:120
#: ../DistUpgrade/DistUpgradeFetcherCore.py:121
msgid "Could not run the upgrade tool"
msgstr "Nun pudo executase la ferramienta d'anovamientu"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:122
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug ubuntu-release-upgrader-core'."
msgstr ""
"Lo más probable ye qu'esto seya un fallu na ferramienta d'anovamientos. "
"Informa d'esto cola orde «ubuntu-bug ubuntu-release-upgrader-core»."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:199
msgid "Upgrade tool signature"
msgstr "Robla de la ferramienta d'anovamientu"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:206
msgid "Upgrade tool"
msgstr "Ferramienta d'anovamientu"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:240
msgid "Failed to fetch"
msgstr "Fallu al descargar"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:241
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr "Falló la baxada del anovamientu. Pue haber un problema cola rede. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:245
msgid "Authentication failed"
msgstr "Falló l'autentificación"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:246
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"Falló l'autentificación de l'anovamientu. Ye dable qu'heba un problema cola "
"rede o col sirvidor. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:251
msgid "Failed to extract"
msgstr "Fallu al sacar"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:252
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"Falló la estraición del anovamientu. Pue haber un problema cola rede o col "
"sirvidor. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:257
msgid "Verification failed"
msgstr "Falló la verificación"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:258
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"Falló la verificación del anovamientu. Pue haber un problema cola rede o col "
"sirvidor. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:272
#: ../DistUpgrade/DistUpgradeFetcherCore.py:278
msgid "Can not run the upgrade"
msgstr "Nun puede executase l'anovamientu"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:273
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"Esto davezu cáusalo un sistema nel que /tmp se montó como non executable. "
"Vuelvi a montalu ensin «noexec» y executa otra vuelta l'anovamientu."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:279
#, python-format
msgid "The error message is '%s'."
msgstr "El mensax de fallu ye «%s»"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:151
msgid "&Upgrade"
msgstr "&Anovar"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:154
#: ../DistUpgrade/DistUpgradeViewKDE.py:715
msgid "&Cancel"
msgstr "En&caboxar"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:156
#: ../data/gtkbuilder/ReleaseNotes.ui.h:1
msgid "Release Notes"
msgstr "Notes de la versión"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:191
msgid "Upgrade"
msgstr "Anovar"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:207
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:222
msgid "Downloading additional package files..."
msgstr "Descargando ficheros de paquetes adicionales..."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:224
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "Ficheru %s de %s a %sB/s"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:228
#, python-format
msgid "File %s of %s"
msgstr "Ficheru %s de %s"

#. print("mediaChange %s %s" % (medium, drive))
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:235
#: ../DistUpgrade/DistUpgradeViewGtk3.py:125
#: ../DistUpgrade/DistUpgradeViewKDE.py:302
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Por favor, inserta '%s' nel dispositivu '%s'"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:237
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:242
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:243
#: ../DistUpgrade/DistUpgradeViewKDE.py:303
msgid "Media Change"
msgstr "Cambéu de preséu"

#: ../DistUpgrade/DistUpgradeQuirks.py:226
msgid "Your graphics hardware may not be fully supported in Ubuntu 14.04."
msgstr ""
"Pue que'l so hardware gráficu nun tenga sofitu completu n'Ubuntu 14.04."

#: ../DistUpgrade/DistUpgradeQuirks.py:228
msgid ""
"Running the 'unity' desktop environment is not fully supported by your "
"graphics hardware. You will maybe end up in a very slow environment after "
"the upgrade. Our advice is to keep the LTS version for now. For more "
"information see https://wiki.ubuntu.com/X/Bugs/"
"UpdateManagerWarningForUnity3D Do you still want to continue with the "
"upgrade?"
msgstr ""
"L'entornu d'escritoriu n'execución «Unity» nun ye compatible dafechu col to "
"hardware de gráficos. Seique termines nun entornu mui lentu dempués del "
"anovamientu. El nuesu conseyu ye caltener por agora la versión LTS. Pa "
"obtener más información, consulta https://wiki.ubuntu.com/X/Bugs/"
"UpdateManagerWarningForUnity3D ¿Aínda quies siguir col anovamientu?"

#: ../DistUpgrade/DistUpgradeQuirks.py:252
msgid "Your graphics hardware may not be fully supported in Ubuntu 12.04 LTS."
msgstr ""
"El hardware de gráficos nun ye compatible dafechu con Ubuntu 12.04 LTS."

#: ../DistUpgrade/DistUpgradeQuirks.py:254
msgid ""
"The support in Ubuntu 12.04 LTS for your Intel graphics hardware is limited "
"and you may encounter problems after the upgrade. For more information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx Do you want to "
"continue with the upgrade?"
msgstr ""
"La compatibilidá n'Ubuntu 12.04 LTS pal hardware de gráficos Intel ta "
"llimitada y pues atopar problemes tres l'anovamientu. Pa tener más "
"información llei https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx "
"¿Quies siguir col anovamientu?"

#: ../DistUpgrade/DistUpgradeQuirks.py:274
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"L'anovamientu pue amenorgar los efeutos d'escritoriu, asina como'l "
"rendimientu de los xuegos y otros programes qu'usen gráficos de mou "
"intensivu."

#: ../DistUpgrade/DistUpgradeQuirks.py:278
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Esti equipu ta usando'l controlador gráficu  «fglrx» de AMD. Nun s'alcuentra "
"disponible n'Ubuntu 10.04 LTS una versión d'esti controlador que funcione "
"col so hardware.\n"
"\n"
"¿Quier continuar?"

#: ../DistUpgrade/DistUpgradeQuirks.py:317
msgid "Sorry, no more upgrades for this system"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:318
#, python-format
msgid ""
"There will not be any further Ubuntu releases for this system's 'i386' "
"architecture.\n"
"\n"
"Updates for Ubuntu %s will continue until %s."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:333
msgid "No ARMv6 CPU"
msgstr "No ARMv6 CPU"

#: ../DistUpgrade/DistUpgradeQuirks.py:334
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"El so sistema usa una CPU ARM que ye más antigua que l'arquiteutura ARMv6. "
"Tolos paquetes en karmic construyéronse con optimizaciones que necesiten de "
"ARMv6 como arquiteutura mínima. Nun ye dable anovar el so sistema a la nueva "
"versión d'Ubuntu con esti hardware."

#: ../DistUpgrade/DistUpgradeQuirks.py:354
msgid "No init available"
msgstr "El degorriu init nun ta disponible"

#: ../DistUpgrade/DistUpgradeQuirks.py:355
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"Paez ser que'l sistema ye un entornu virtualizáu ensin un degorriu init (un "
"Linux-VServer, p.ex.). Ubuntu 10.04 LTS nun puede furrular nesta triba "
"d'entornu, polo que primero requierse un anovamientu de la configuración de "
"la to máquina virtual.\n"
"\n"
"¿Daveres que quies siguir?"

#: ../DistUpgrade/DistUpgradeQuirks.py:396
msgid "The Robot Operating System (ROS) is installed"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:397
msgid ""
"It appears that ROS is currently installed. Each ROS release is very strict "
"about the versions of Ubuntu it supports, and Ubuntu upgrades can fail if "
"that guidance isn't followed. Before continuing, please either uninstall "
"ROS, or ensure the ROS release you have installed supports the version of "
"Ubuntu to which you're upgrading.\n"
"\n"
"For ROS 1 releases, refer to REP 3:\n"
"https://www.ros.org/reps/rep-0003.html\n"
"\n"
"For ROS 2 releases, refer to REP 2000:\n"
"https://www.ros.org/reps/rep-2000.html\n"
"\n"
"Are you sure you want to continue?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:515
msgid "Connection to the Snap Store failed"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:516
msgid ""
"You have the package lxd installed but your system is unable to reach the "
"Snap Store. lxd is now provided via a snap and the release upgrade will fail "
"if snapd is not functional. Please make sure you're connected to the "
"Internet and update any firewall or proxy settings as needed so that you can "
"reach api.snapcraft.io. If you are an enterprise with a firewall setup you "
"may want to configure a Snap Store proxy."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:531
msgid "Connection to Snap Store failed"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:532
msgid ""
"Your system does not have a connection to the Snap Store. For the best "
"upgrade experience make sure that your system can connect to api.snapcraft."
"io.\n"
"Do you still want to continue with the upgrade?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:541
msgid "Outdated snapd package"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:542
msgid ""
"Your system does not have the latest version of snapd. Please update the "
"version of snapd on your system to improve the upgrade experience.\n"
"Do you still want to continue with the upgrade?"
msgstr ""

#. now perform direct API calls to the store, requesting size
#. information for each of the snaps needing installation
#: ../DistUpgrade/DistUpgradeQuirks.py:565
msgid "Calculating snap size requirements"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:596
msgid "Processing snap replacements"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:602
#, python-format
msgid "refreshing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:606
#, python-format
msgid "removing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:609
#, python-format
msgid "installing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:638
msgid "PAE not enabled"
msgstr "PAE nun ta activáu"

#: ../DistUpgrade/DistUpgradeQuirks.py:639
msgid ""
"Your system uses a CPU that does not have PAE enabled. Ubuntu only supports "
"non-PAE systems up to Ubuntu 12.04. To upgrade to a later version of Ubuntu, "
"you must enable PAE (if this is possible) see:\n"
"http://help.ubuntu.com/community/EnablingPAE"
msgstr ""
"El so sistema utiliza una CPU que nun tien activáu PAE. Ubuntu sólo ye "
"compatible con sistemes non-PAE fasta Ubuntu 12.04. P'anovar a una versión "
"más reciente d'Ubuntu, tien d'activar PAE (si ye posible) vea:\n"
" http://help.ubuntu.com/community/EnablingPAE"

#: ../DistUpgrade/DistUpgradeQuirks.py:1000
msgid "Checking for installed snaps"
msgstr ""

#: ../DistUpgrade/DistUpgradeMain.py:58
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr "Usar la ruta pa guetar un cdrom con paquetes d'anovación"

#: ../DistUpgrade/DistUpgradeMain.py:64
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Usar interfaz d'usuariu. Anguaño disponibles: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:67
msgid "*DEPRECATED* this option will be ignored"
msgstr "*OBSOLETU* esta opción va inorase"

#: ../DistUpgrade/DistUpgradeMain.py:70
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr ""
"Facer namái un anovamientu parcial (nun se reescribirá el «sources.list»)"

#: ../DistUpgrade/DistUpgradeMain.py:73
msgid "Disable GNU screen support"
msgstr "Desactiva'l sofitu de pantalla GNU"

#: ../DistUpgrade/DistUpgradeMain.py:75
msgid "Set datadir"
msgstr "Afitar datadir"

#: ../DistUpgrade/DistUpgradeMain.py:78
msgid "Upgrade to the development release"
msgstr "Anovar a la versión en desendolcu"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:146
#: ../DistUpgrade/DistUpgradeViewKDE.py:316
msgid "Fetching is complete"
msgstr "La descarga completóse"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:161
#: ../DistUpgrade/DistUpgradeViewKDE.py:332
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "Descargando ficheru %li de %li a %sB/s"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:163
#: ../DistUpgrade/DistUpgradeViewGtk3.py:325
#: ../DistUpgrade/DistUpgradeViewKDE.py:333
#: ../DistUpgrade/DistUpgradeViewKDE.py:483
#, python-format
msgid "About %s remaining"
msgstr "Falten al rodiu de %s"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:166
#: ../DistUpgrade/DistUpgradeViewKDE.py:335
#, python-format
msgid "Fetching file %li of %li"
msgstr "Descargando ficheru %li de %li"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk3.py:198
#: ../DistUpgrade/DistUpgradeViewKDE.py:373
msgid "Applying changes"
msgstr "Aplicando cambeos"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk3.py:224
#: ../DistUpgrade/DistUpgradeViewKDE.py:386
msgid "dependency problems - leaving unconfigured"
msgstr "problemes de dependencies - déxase ensin configurar"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:229
#: ../DistUpgrade/DistUpgradeViewKDE.py:388
#, python-format
msgid "Could not install '%s'"
msgstr "Nun puede istalase '%s'"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:230
#: ../DistUpgrade/DistUpgradeViewKDE.py:389
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"L'anovamientu continuará, pero'l paquete '%s', nun paez tar en bon estáu. "
"Por favor, considera unviar un informe col fallu tocante a esti problema."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk3.py:247
#: ../DistUpgrade/DistUpgradeViewKDE.py:411
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr "¿Deseya sustituyir el ficheru de configuración modificáu«%s»?"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:248
#: ../DistUpgrade/DistUpgradeViewKDE.py:412
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Perderá tolos cambeos que tenga fecho nesi ficheru de configuración si "
"decide sustituyilu por una nueva versión."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:268
#: ../DistUpgrade/DistUpgradeViewKDE.py:435
msgid "The 'diff' command was not found"
msgstr "El comandu 'diff' nun fue atopau"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:535
#: ../DistUpgrade/DistUpgradeViewText.py:140
msgid "A fatal error occurred"
msgstr "Ocurrió un error fatal"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:536
msgid ""
"Please report this as a bug (if you haven't already) and include the files /"
"var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Informa d'esti fallu (si aínda nun lo fexesti) ya inclúi los ficheros /var/"
"log/dist-upgrade/main.log y /var/log/dist-upgrade/apt.log nel informe. "
"L'anovamientu encaboxóse.\n"
"El ficheru sources.list orixinal guardóse en /etc/apt/sources.list."
"distUpgrade."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:566
msgid "Ctrl-c pressed"
msgstr "Calcóse Ctrl-C"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:567
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"Esto encaboxará la operación y puede dexar al sistema nun estáu defeutuosu. "
"¿Daveres que quier facer eso?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk3.py:706
msgid "To prevent data loss close all open applications and documents."
msgstr ""
"Pa prevenir la perda de datos, zarra toles aplicaciones y documentos "
"abiertos."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:720
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr "Yá nun ta sofitáu téunicamente por Canonical (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:721
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr "<b>Desanovar (%s)</b>"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:722
#, python-format
msgid "Remove (%s)"
msgstr "Desaniciar (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:723
#, python-format
msgid "No longer needed (%s)"
msgstr "Yá nun fai falta (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:724
#, python-format
msgid "Install (%s)"
msgstr "Instalar (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:725
#, python-format
msgid "Upgrade (%s)"
msgstr "Anovar (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:773
msgid "Cancel"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk3.py:774
#: ../DistUpgrade/DistUpgradeViewKDE.py:939
msgid "Continue"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewKDE.py:447
msgid "Show Difference >>>"
msgstr "Amosar diferencies >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:450
msgid "<<< Hide Difference"
msgstr "<<< Soverar diferencies"

#: ../DistUpgrade/DistUpgradeViewKDE.py:701
msgid "Error"
msgstr "Fallu"

#: ../DistUpgrade/DistUpgradeViewKDE.py:719
#: ../DistUpgrade/DistUpgradeViewKDE.py:960
msgid "&Close"
msgstr "&Zarrar"

#: ../DistUpgrade/DistUpgradeViewKDE.py:753
msgid "Show Terminal >>>"
msgstr "Amosar terminal >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:756
msgid "<<< Hide Terminal"
msgstr "<<< Soverar terminal"

#: ../DistUpgrade/DistUpgradeViewKDE.py:829
msgid "Information"
msgstr "Información"

#: ../DistUpgrade/DistUpgradeViewKDE.py:877
msgid "&Start Upgrade"
msgstr "&Aniciar anovamientu"

#: ../DistUpgrade/DistUpgradeViewKDE.py:879
#: ../DistUpgrade/DistUpgradeViewKDE.py:923
#: ../DistUpgrade/DistUpgradeViewKDE.py:926
#: ../data/gtkbuilder/DistUpgrade.ui.h:7
msgid "Details"
msgstr "Detalles"

#: ../DistUpgrade/DistUpgradeViewKDE.py:901
#, python-format
msgid "No longer supported %s"
msgstr "Yá nun ta sofitáu %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:903
#, python-format
msgid "Remove %s"
msgstr "Esaniciar %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:905
#: ../DistUpgrade/DistUpgradeViewText.py:239
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "Desaniciar (autoinstalóse) %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:907
#, python-format
msgid "Install %s"
msgstr "Instalar %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:909
#, python-format
msgid "Upgrade %s"
msgstr "Anovar %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:956
#: ../DistUpgrade/DistUpgradeViewText.py:292
msgid "Restart required"
msgstr "Necesítase reaniciar"

#: ../DistUpgrade/DistUpgradeViewKDE.py:956
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>Reanicia'l sistema pa completar l'anovamientu</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:959
#: ../data/gtkbuilder/DistUpgrade.ui.h:14
msgid "_Restart Now"
msgstr "_Reaniciar agora"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:977
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>¿Encaboxar l'anovamientu en cursu?</big></b>\n"
"\n"
"El sistema podría quedar nun estáu non usable si encaboxa l'anovamientu. "
"Encamentámos-y que siga col anovamientu."

#: ../DistUpgrade/DistUpgradeViewKDE.py:981
msgid "Cancel Upgrade?"
msgstr "¿Encaboxar l'anovamientu?"

#: ../DistUpgrade/DistUpgradeView.py:82
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li día"
msgstr[1] "%li díes"

#: ../DistUpgrade/DistUpgradeView.py:84
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li hora"
msgstr[1] "%li hores"

#: ../DistUpgrade/DistUpgradeView.py:86
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li minutu"
msgstr[1] "%li minutos"

#: ../DistUpgrade/DistUpgradeView.py:87
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li segundu"
msgstr[1] "%li segundos"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:103
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:121
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:172
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"Esta descarga va llevar %s aproximadamente con una conexón DSL de 1Mbit y %s "
"aproximadamente con un módem de 56k."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:176
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "Esta descarga va tardar aproximadamente %s cola conexón actual. "

#. Declare these translatable strings from the .ui files here so that
#. xgettext picks them up.
#: ../DistUpgrade/DistUpgradeView.py:270 ../data/gtkbuilder/DistUpgrade.ui.h:22
msgid "Preparing to upgrade"
msgstr "Tresnando l'anovamientu"

#: ../DistUpgrade/DistUpgradeView.py:271
msgid "Getting new software channels"
msgstr "Obteniendo nuevos canales de software"

#: ../DistUpgrade/DistUpgradeView.py:272 ../data/gtkbuilder/DistUpgrade.ui.h:24
msgid "Getting new packages"
msgstr "Obteniendo paquetes nuevos"

#: ../DistUpgrade/DistUpgradeView.py:273 ../data/gtkbuilder/DistUpgrade.ui.h:27
msgid "Installing the upgrades"
msgstr "Instalando los anovamientos"

#: ../DistUpgrade/DistUpgradeView.py:274 ../data/gtkbuilder/DistUpgrade.ui.h:26
msgid "Cleaning up"
msgstr "Llimpiando"

#: ../DistUpgrade/DistUpgradeView.py:363
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
"Hai %(amount)d paquete instaláu que yá nun ta sofitáu por Canonical. Pues "
"siguir obteniendo sofitu de la comunidá."
msgstr[1] ""
"Hai %(amount)d paquetes instalaos que yá nun tán sofitaos por Canonical. "
"Pues siguir obteniendo sofitu de la comunidá."

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:372
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "Va desinstalase %d paquete."
msgstr[1] "Van desinstalase %d paquetes."

#: ../DistUpgrade/DistUpgradeView.py:377
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "Va instalase %d paquete nuevu."
msgstr[1] "Van instalase %d paquetes nuevos."

#: ../DistUpgrade/DistUpgradeView.py:383
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "Va a anovase %d paquete."
msgstr[1] "Van a anovase %d paquetes."

#: ../DistUpgrade/DistUpgradeView.py:391
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"Tienes de descargar un total de %s. "

#: ../DistUpgrade/DistUpgradeView.py:396
msgid ""
"Installing the upgrade can take several hours. Once the download has "
"finished, the process cannot be canceled."
msgstr ""
"Esti anovamientu pue llevar delles hores. Una vegada fine la descarga, el "
"procesu nun pue encaboxase."

#: ../DistUpgrade/DistUpgradeView.py:400
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be canceled."
msgstr ""
"Obtener ya instalar l'anovamientu pue llevar delles hores. Una vegada que "
"fine la descarga, el procesu nun pue encaboxase."

#: ../DistUpgrade/DistUpgradeView.py:405
msgid "Removing the packages can take several hours. "
msgstr "La desinstalación de los paquetes pue llevar delles hores. "

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:410
msgid "The software on this computer is up to date."
msgstr "El software d'esti equipu ta anováu"

#: ../DistUpgrade/DistUpgradeView.py:411
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr ""
"Nun hai anovamientos disponibles pal sistema. Encaboxóse l'anovamientu."

#: ../DistUpgrade/DistUpgradeView.py:427
msgid "Reboot required"
msgstr "Necesítase reaniciar"

#: ../DistUpgrade/DistUpgradeView.py:428
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr ""
"L'anovamientu finó y necesítase reaniciar l'equipu. ¿Quies facelo agora?"

#: ../DistUpgrade/DistUpgradeViewText.py:82
#, python-format
msgid "Progress: [%3i%%]"
msgstr "Progresu: [%3i%%]"

#: ../DistUpgrade/DistUpgradeViewText.py:141
msgid ""
"Please report this as a bug and include the files /var/log/dist-upgrade/main."
"log and /var/log/dist-upgrade/apt.log in your report. The upgrade has "
"aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Informa d'esti fallu ya inclúi los ficheros /var/log/dist-upgrade/main.log "
"y /var/log/dist-upgrade/apt.log nel informe. L'anovamientu encaboxóse.\n"
"El ficheru sources.list original atroxóse en /etc/apt/sources.list."
"distUpgrade."

#: ../DistUpgrade/DistUpgradeViewText.py:165
msgid "Aborting"
msgstr "Albortando"

#: ../DistUpgrade/DistUpgradeViewText.py:171
msgid "Demoted:\n"
msgstr "Pa quitar:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:178
msgid "To continue please press [ENTER]"
msgstr "Pa continuar, calca Intro"

#: ../DistUpgrade/DistUpgradeViewText.py:215
#: ../DistUpgrade/DistUpgradeViewText.py:253
#: ../DistUpgrade/DistUpgradeViewText.py:261
msgid "Continue [yN] "
msgstr "Siguir [sN] "

#: ../DistUpgrade/DistUpgradeViewText.py:215
#: ../DistUpgrade/DistUpgradeViewText.py:253
msgid "Details [d]"
msgstr "Detalles [d]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:219
#: ../DistUpgrade/DistUpgradeViewText.py:264
msgid "y"
msgstr "s"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:222
#: ../DistUpgrade/DistUpgradeViewText.py:271
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:225
msgid "d"
msgstr "d"

#: ../DistUpgrade/DistUpgradeViewText.py:230
#, python-format
msgid "No longer supported: %s\n"
msgstr "Yá nun ta sofitáu: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:235
#, python-format
msgid "Remove: %s\n"
msgstr "Quitar: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:245
#, python-format
msgid "Install: %s\n"
msgstr "Instalar: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:250
#, python-format
msgid "Upgrade: %s\n"
msgstr "Anovar: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:268
msgid "Continue [Yn] "
msgstr "Continuar [Sn] "

#: ../DistUpgrade/DistUpgradeViewText.py:293
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"Pa finar l'anovamientu necesítase reaniciar.\n"
"Si escueyes «s» el sistema reaniciaráse."

#: ../DistUpgrade/GtkProgress.py:71
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "Descargando ficheru %(current)li de %(total)li a %(speed)s/s"

#: ../DistUpgrade/GtkProgress.py:77
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "Descargando ficheru %(current)li de %(total)li"

#: ../data/gtkbuilder/AcquireProgress.ui.h:1
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
msgid "Show progress of individual files"
msgstr "Amosar el progresu de cada ficheru individual"

#: ../data/gtkbuilder/DistUpgrade.ui.h:1
msgid "_Cancel Upgrade"
msgstr "_Encaboxar l'anovamientu"

#: ../data/gtkbuilder/DistUpgrade.ui.h:2
msgid "_Resume Upgrade"
msgstr "_Siguir col anovamientu"

#: ../data/gtkbuilder/DistUpgrade.ui.h:3
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>¿Encaboxar l'anovamientu en cursu?</big></b>\n"
"\n"
"El sistema podría quedar nun estáu non usable si encaboxes l'anovamientu. "
"Encamentámoste que sigas col anovamientu."

#: ../data/gtkbuilder/DistUpgrade.ui.h:6
msgid "_Start Upgrade"
msgstr "_Aniciar l'anovamientu"

#: ../data/gtkbuilder/DistUpgrade.ui.h:9
msgid "_Replace"
msgstr "_Trocar"

#: ../data/gtkbuilder/DistUpgrade.ui.h:10
msgid "Difference between the files"
msgstr "Diferencia ente los ficheros"

#: ../data/gtkbuilder/DistUpgrade.ui.h:11
msgid "_Report Bug"
msgstr "_Informar d'un fallu"

#: ../data/gtkbuilder/DistUpgrade.ui.h:12
msgid "_Continue"
msgstr "_Siguir"

#: ../data/gtkbuilder/DistUpgrade.ui.h:13
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>¿Aniciar l'anovamientu?</big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:15
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>Reanicia'l sistema pa completar l'anovamientu</big></b>\n"
"\n"
"Por favor guarda tolos trabayos enantes de continuar."

#: ../data/gtkbuilder/DistUpgrade.ui.h:18
msgid "Distribution Upgrade"
msgstr "Anovamientu de la distribución"

#: ../data/gtkbuilder/DistUpgrade.ui.h:20
#, no-c-format
msgid "<b><big>Upgrading Ubuntu to version %s</big></b>"
msgstr "<b><big>Anovando Ubuntu a la versión %s</big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:21
msgid "    "
msgstr "    "

#: ../data/gtkbuilder/DistUpgrade.ui.h:23
msgid "Setting new software channels"
msgstr "Configurando nuevos canales de software"

#: ../data/gtkbuilder/DistUpgrade.ui.h:25
msgid "Restarting the computer"
msgstr "Reaniciando l'equipu"

#: ../data/gtkbuilder/DistUpgrade.ui.h:28
msgid "Terminal"
msgstr "Terminal"

#: ../data/gtkbuilder/ReleaseNotes.ui.h:2
msgid "_Upgrade"
msgstr "_Anovar"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid "<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr "<b>Hai disponible una nueva versión d'Ubuntu. ¿Prestaríate anovar?</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Don't Upgrade"
msgstr "Non anovar"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Ask Me Later"
msgstr "Entrugar más sero"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Yes, Upgrade Now"
msgstr "Sí, anovar agora"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "Decidisti non anovar a la nueva versión d'Ubuntu"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid ""
"You can upgrade at a later time by opening Software Updater and click on "
"\"Upgrade\"."
msgstr ""
"Pues anovaar más sero diendo al Anovador de software y calcando n'«Anovar»."

#: ../data/com.ubuntu.release-upgrader.policy.in.h:1
msgid "Perform a release upgrade"
msgstr "Facer un anovamientu de la versión"

#: ../data/com.ubuntu.release-upgrader.policy.in.h:2
msgid "To upgrade Ubuntu, you need to authenticate."
msgstr "P'anovar Ubuntu, necesites autenticate."

#: ../data/com.ubuntu.release-upgrader.policy.in.h:3
msgid "Perform a partial upgrade"
msgstr "Facer un anovamientu parcial"

#: ../data/com.ubuntu.release-upgrader.policy.in.h:4
msgid "To perform a partial upgrade, you need to authenticate."
msgstr "Pa facer un anovamientu parcial, necesites autenticate."

#: ../do-partial-upgrade:75 ../do-release-upgrade:80
msgid "Show version and exit"
msgstr "Amosar la versión y salir"

#: ../do-partial-upgrade:78 ../do-release-upgrade:87
msgid "Directory that contains the data files"
msgstr "Direutoriu que contién los ficheros de datos"

#: ../do-partial-upgrade:81 ../do-release-upgrade:100
msgid "Run the specified frontend"
msgstr "Executar la interface especificada"

#: ../do-partial-upgrade:112
msgid "Running partial upgrade"
msgstr "Executando un anovamientu parcial"

#: ../do-release-upgrade:34 ../do-release-upgrade:46
msgid "Downloading the release upgrade tool"
msgstr "Descargar la ferramienta d'anovamientu del llanzamientu"

#: ../do-release-upgrade:83
msgid ""
"If using the latest supported release, upgrade to the development release"
msgstr ""

#: ../do-release-upgrade:90 ../check_new_release_gtk.py:181
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""
"Intenta anovar a la cabera versión usando l'anovador de $distro-proposed"

#: ../do-release-upgrade:94
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"Executar nun mou especial d'anovamientu.\n"
"Anguaño sofiténse los moos «desktop» (p'anovamientos normales d'un sistema "
"d'escritoriu) y «server» (pa sirvidores)."

#: ../do-release-upgrade:103
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr ""
"Namái comprueba si ta disponible una nueva versión de la distribución ya "
"informa del resultáu per aciu d'un códigu de salida"

#: ../do-release-upgrade:108
msgid ""
"Try the upgrade with third party mirrors and repositories enabled instead of "
"commenting them out."
msgstr ""

#: ../do-release-upgrade:122
msgid "The options --devel-release and --proposed are"
msgstr "Les opciones  --devel-release y --proposed son"

#: ../do-release-upgrade:123
msgid "mutually exclusive. Please use only one of them."
msgstr "mutuamente esclusives. Use sólo una d'elles."

#: ../do-release-upgrade:127
msgid "Checking for a new Ubuntu release"
msgstr "Comprobar si hai una versión nueva d'Ubuntu"

#: ../do-release-upgrade:139
msgid "Your Ubuntu release is not supported anymore."
msgstr "La to versión d'Ubuntu yá nun tien sofitu téunicu."

#: ../do-release-upgrade:140
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""
"Pa saber más tocante a esti anovamientu, visita:\n"
"%(url)s\n"

#: ../do-release-upgrade:147
msgid "In /etc/update-manager/release-upgrades Prompt "
msgstr ""

#: ../do-release-upgrade:148
msgid "is set to never so upgrading is not possible."
msgstr ""

#: ../do-release-upgrade:150
msgid "There is no development version of an LTS available."
msgstr ""

#: ../do-release-upgrade:151
msgid "To upgrade to the latest non-LTS development release "
msgstr ""

#: ../do-release-upgrade:152
msgid "set Prompt=normal in /etc/update-manager/release-upgrades."
msgstr ""

#: ../do-release-upgrade:154
msgid "Upgrades to the development release are only "
msgstr ""

#: ../do-release-upgrade:155
msgid "available from the latest supported release."
msgstr ""

#: ../do-release-upgrade:157
msgid "No new release found."
msgstr ""

#: ../do-release-upgrade:162
msgid "Release upgrade not possible right now"
msgstr "L'anovamientu de versión nun ye posible nesti intre"

#: ../do-release-upgrade:163
#, c-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr ""
"L'anovamientu de versión nun ye posible nesti intre, inténtalo más sero. El "
"sirvidor informó: «%s»"

#: ../do-release-upgrade:169
#, c-format
msgid "New release '%s' available."
msgstr "Ta disponible la nueva versión «%s»."

#: ../do-release-upgrade:170
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "Executar 'do-release-upgrade' p'anovase a élli."

#: ../do-release-upgrade:187
msgid "Please install all available updates for your release before upgrading."
msgstr ""

#: ../do-release-upgrade:203
msgid ""
"You have not rebooted after updating a package which requires a reboot. "
"Please reboot before upgrading."
msgstr ""

#: ../check_new_release_gtk.py:85
msgid "Software Updater"
msgstr "Anovamientu de software"

#: ../check_new_release_gtk.py:113
#, python-format
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "Ta disponible l'anovamientu de versión a Ubuntu %(version)s"

#. print("don't upgrade")
#: ../check_new_release_gtk.py:144
#, python-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "Decidisti nun anovar a Ubuntu %s"

#: ../check_new_release_gtk.py:177
msgid "Check if upgrading to the latest devel release is possible"
msgstr "Comprobar si ye dable anovar a la cabera versión de desendolcu"

#: ../check_new_release_gtk.py:186
msgid "Add debug output"
msgstr "Amestar resultáu de la depuración"

#~ msgid ""
#~ "An unresolvable problem occurred while calculating the upgrade:\n"
#~ "%s\n"
#~ "\n"
#~ " This can be caused by:\n"
#~ " * Upgrading to a pre-release version of Ubuntu\n"
#~ " * Running the current pre-release version of Ubuntu\n"
#~ " * Unofficial software packages not provided by Ubuntu\n"
#~ "\n"
#~ msgstr ""
#~ "Hebo un problema ensin solución al calcular l'anovamientu:\n"
#~ "%s\n"
#~ "\n"
#~ " L'orixe pue ser por cuenta de:\n"
#~ " * Anovar a una versión pre-llanzamientu d'Ubuntu\n"
#~ " * Tener yá instalada la versión pre-llanzamientu actual d'Ubuntu\n"
#~ " * Paquetes de software non oficiales que Ubuntu nun ufre\n"
#~ "\n"

#~ msgid ""
#~ "Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
#~ "desktop or edubuntu-desktop package and it was not possible to detect "
#~ "which version of Ubuntu you are running.\n"
#~ " Please install one of the packages above first using synaptic or apt-get "
#~ "before proceeding."
#~ msgstr ""
#~ "El to sistema nun contién un paquete ubuntu-desktop, kubuntu-desktop, "
#~ "xubuntu-desktop o edubuntu-desktop y nun foi dable detectar qué versión "
#~ "d'Ubuntu tas executando.\n"
#~ " Por favor, instala un de los paquetes d'abaxo enantes d'usar Synaptic o "
#~ "apt-get."

#~ msgid "Sandbox setup failed"
#~ msgstr "Falló la configuración de Sandbox"

#~ msgid "It was not possible to create the sandbox environment."
#~ msgstr "Nun foi dable crear un entornu sandbox."

#~ msgid "Sandbox mode"
#~ msgstr "Mou Sandbox"

#~ msgid ""
#~ "This upgrade is running in sandbox (test) mode. All changes are written "
#~ "to '%s' and will be lost on the next reboot.\n"
#~ "\n"
#~ "*No* changes written to a system directory from now until the next reboot "
#~ "are permanent."
#~ msgstr ""
#~ "Esti anovamientu ta faciéndose nel  mou (prueba) «sandbox». Tolos cambeos "
#~ "escríbense en «%s» y van perdese nel siguiente arranque.\n"
#~ "\n"
#~ "*Dengún* de los cambeos escritos nel direutoriu de sistema va ser "
#~ "permanente, dende agora hasta'l siguiente arranque."

#~ msgid ""
#~ "Your python install is corrupted. Please fix the '/usr/bin/python' "
#~ "symlink."
#~ msgstr ""
#~ "La so instalación de python ta toyida. Por favor, igüe l'enllaz simbólicu "
#~ "«/usr/bin/python»."

#~ msgid "Package 'debsig-verify' is installed"
#~ msgstr "El paquete 'debsig-verify' ta instaláu"

#~ msgid ""
#~ "The upgrade can not continue with that package installed.\n"
#~ "Please remove it with synaptic or 'apt-get remove debsig-verify' first "
#~ "and run the upgrade again."
#~ msgstr ""
#~ "L'anovamientu nun pue continuar con esi paquete instaláu.\n"
#~ "Desinstálalu con synaptic o «apt-get remove debsig-verify» primero y "
#~ "executa l'anovamientu otra vegada."

#~ msgid ""
#~ "While scanning your repository information no mirror entry for the "
#~ "upgrade was found. This can happen if you run a internal mirror or if the "
#~ "mirror information is out of date.\n"
#~ "\n"
#~ "Do you want to rewrite your 'sources.list' file anyway? If you choose "
#~ "'Yes' here it will update all '%s' to '%s' entries.\n"
#~ "If you select 'No' the upgrade will cancel."
#~ msgstr ""
#~ "Mientres s'esploraba la información del so repositoriu, nun s'alcontró "
#~ "denguna entrada pa la réplica de l'anovamientu. Esto puede ocurrir si "
#~ "cuerre una réplica interna o si la información de la réplica ye antigua.\n"
#~ "\n"
#~ "¿Deseya reescribir el so ficheru «sources.list» de toes formes? Si "
#~ "escueye «Sí» anovaránse toles entraes «%s» a «%s».\n"
#~ "Si escueye «Non» encaboxaráse l'anovamientu."

#~ msgid ""
#~ "The upgrade has aborted. The upgrade needs a total of %s free space on "
#~ "disk '%s'. Please free at least an additional %s of disk space on '%s'. "
#~ "Empty your trash and remove temporary packages of former installations "
#~ "using 'sudo apt-get clean'."
#~ msgstr ""
#~ "Encaboxóse l'anovamientu. L'anovamientu necesita un total de %s d'espaciu "
#~ "llibre nel discu «%s». Lliber a lo menos %s d'espaciu nel discu «%s». "
#~ "Prueba vaciando la papelera y desaniciando paquetes temporales d'antigües "
#~ "instalaciones usando la orde «sudo apt-get clean»."

#~ msgid "evms in use"
#~ msgstr "Ta usándose «evms»"

#~ msgid ""
#~ "Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
#~ "software is no longer supported, please switch it off and run the upgrade "
#~ "again when this is done."
#~ msgstr ""
#~ "El to sistema ta usando'l xestor de volúmenes «evms» en /proc/mounts. El "
#~ "software «evms» yá nun ta sofitáu; por favor, desactívalu y torna a "
#~ "executar de nuevo l'anovamientu."

#~ msgid ""
#~ "This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
#~ "version of this driver is available that works with your video card in "
#~ "Ubuntu 10.04 LTS.\n"
#~ "\n"
#~ "Do you want to continue?"
#~ msgstr ""
#~ "Esti equipu ta usando'l controlador gráficu  «nvidia» de NVIDIA. Nun "
#~ "s'alcuentra disponible n'Ubuntu 10.04 LTS una versión d'esti controlador "
#~ "que funcione col so hardware.\n"
#~ "\n"
#~ "¿Quier continuar?"

#~ msgid "No i686 CPU"
#~ msgstr "La CPU nun ye i686"

#~ msgid ""
#~ "Your system uses an i586 CPU or a CPU that does not have the 'cmov' "
#~ "extension. All packages were built with optimizations requiring i686 as "
#~ "the minimal architecture. It is not possible to upgrade your system to a "
#~ "new Ubuntu release with this hardware."
#~ msgstr ""
#~ "El sistema usa una CPU i586 o una CPU que nun tien la estensión «cmov». "
#~ "Construyéronse tolos paquetes con meyores que necesiten un i686 como "
#~ "arquiteutura mínima. Nun ye posible anovar el sistema a una versión nueva "
#~ "d'Ubuntu con esti hardware."

#~ msgid "Sandbox upgrade using aufs"
#~ msgstr "Anovamientu de prueba usando aufs"

#~ msgid "Test upgrade with a sandbox aufs overlay"
#~ msgstr "Comprobar l'anovamientu nuna capa aufs de caxa de sable (sandbox)"

#~ msgid "No new release found"
#~ msgstr "Nun s'alcontró denguna edición nueva"

#~ msgid "Authentication is required to perform a partial upgrade"
#~ msgstr "Pa facer un anovamientu parcial, necesites autenticate"

#~ msgid "Authentication is required to perform a release upgrade"
#~ msgstr "P'anovar la distribución, necesites autenticate"

#~ msgid ""
#~ "If none of this applies, then please report this bug using the command "
#~ "'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
#~ msgstr ""
#~ "Si nada d'esto aplica, informa d'esti fallu usando la orde «ubuntu-bug "
#~ "ubuntu-release-upgrader-core» nuna terminal."

#~ msgid ""
#~ "An unresolvable problem occurred while calculating the upgrade.\n"
#~ "\n"
#~ " This can be caused by:\n"
#~ " * Upgrading to a pre-release version of Ubuntu\n"
#~ " * Running the current pre-release version of Ubuntu\n"
#~ " * Unofficial software packages not provided by Ubuntu\n"
#~ "\n"
#~ msgstr ""
#~ "Hebo un fallu mientres se calculaba l'anovamientu.\n"
#~ "\n"
#~ " Esto pudo debese por:\n"
#~ " * Anovar a una versión pre-llanzamientu d'Ubuntu\n"
#~ " * Tar executando la versión pre-llanzamientu actual d'Ubuntu\n"
#~ " * Paquetes non oficiales de software non proporcionaos por Ubuntu\n"
#~ "\n"

#~ msgid "<b><big>Upgrading Ubuntu to version 15.04</big></b>"
#~ msgstr "<b><big>Anovando Ubuntu a la versión 15.04</big></b>"
